tests/python/contrib/test_hexagon/infrastructure.py

```diff
@@ -347,7 +347,12 @@ def quantize_np(arr_np: numpy.ndarray, dtype: str):
     elif fmin > 0:
         fmin = 0.0
 
-    scale = (fmax - fmin) / (qmax - qmin)
-    zero_point = numpy.rint((fmax * qmin - fmin * qmax) / (fmax - fmin)).astype("int32")
+    if fmax == fmin:
+        scale = fmax
+        zero_point = 0
+    else:
```

Comment:
Do you really need this?  The following code will make sure that either fmax or fmin are set to 0 which means that unless fmax and fmin are 0, they can't be equal.  
   if fmax < 0:
        fmax = 0.0
    elif fmin > 0:
        fmin = 0.0